docs/user-guide.md

```
@@ -98,26 +98,35 @@ This creates:
+It also auto-installs stop hooks for Claude Code and Cursor (if selected) and installs the `/gauntlet-setup` skill.
 
-Create YAML files under `.gauntlet/checks/`, e.g. `.gauntlet/checks/lint.yml`:
+### 2) Configure checks and reviews
 
-```yaml
-name: lint
-command: bun test
-working_directory: .
+Run the `/gauntlet-setup` skill in your AI agent session:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)